packages/next/server/web/spec-extension/request.ts

```diff
@@ -56,10 +55,7 @@ export class NextRequest extends Request {
   }
 
   public get page() {
```

Comment:
Do we want to add ts comment `@deprecated` notion for this? so it's known as deprecated through typing

Reply:
Oh, good point; I never did that before, let me open a new PR 🙂

Reply:
it's being mentioned as deprecated in runtime but I think the text should be strikethrough.

![image](https://user-images.githubusercontent.com/46083126/176639570-7bcbec0b-ecf9-48a4-b39d-de987d432c04.png)

not sure why that is still exported since we can't use it anymore in next12.2

and also I have a question. what will be the good replacement for this `req.page.name`
in my case, I want to get the dynamic routes folder name like `pages/[customer]`. I used to do this with `req.page.name`